.travis.yml

```diff
@@ -23,17 +25,17 @@ script:
 
 after_success:
   - codecov
+  - if [[ $BUILD_INSTALLER == 'yes' ]]; then pip install pynsist; ./script/makeinstaller.sh; fi
 
-before_deploy:
-  - sudo apt-get install nsis
-  - sudo pip install s3cmd
-  - pip install pynsist
-  - "./script/makeinstaller.sh"
+addons:
+  apt:
+    packages:
+    - nsis
 
 deploy:
   provider: script
   skip_cleanup: true
-  script: if [ -n "${TRAVIS_TAG}" ]; then s3cmd put build/nsis/streamlink-*.exe s3://streamlink-builds/release/windows/; else s3cmd put build/nsis/streamlink-*.exe s3://streamlink-builds/nightly/windows/; s3cmd put build/nsis/streamlink-*.exe s3://streamlink-builds/nightly/windows/streamlink-latest.exe; fi
+  script: sudo pip install s3cmd; if [ -n "${TRAVIS_TAG}" ]; then s3cmd put build/nsis/streamlink-*.exe s3://streamlink-builds/release/windows/; else s3cmd put build/nsis/streamlink-*.exe s3://streamlink-builds/nightly/windows/; s3cmd put build/nsis/streamlink-*.exe s3://streamlink-builds/nightly/windows/streamlink-latest.exe; fi
```

Comment:
> s3cmd doesn't work with Python 3, and is only required when there is a deploy to S3

Yes, right, but the installer gets built for every commit, so it's better to bundle all deps IMO.

> I couldn't get it to upload only small set of files

Isn't it possible to use the [`local_dir`](https://docs.travis-ci.com/user/deployment/s3/#Deploy-From-Only-One-Folder) s3 deploy property? I haven't used s3 before, though, so maybe I'm wrong.

> also I wanted to to copy the current installer to streamlink-latest.exe

Files could me moved/renamed by the bash script